cpp/include/nvtext/replace.hpp

```diff
@@ -82,7 +82,7 @@ namespace CUDF_EXPORT nvtext {
  *                  The default of empty string will identify tokens using whitespace.
  * @param stream CUDA stream used for device memory operations and kernel launches
  * @param mr Device memory resource used to allocate the returned column's device memory
- * @return New strings columns of with replaced strings
+ * @return New strings column of replaced strings
```

Comment:
Seems like "with" makes more sense than "of" here, since the non-replaced strings are included too.